src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1569,18 +1569,29 @@ core::Error runGrepOperation(const GrepOptions& grepOptions, const ReplaceOption
    return Success();
 }
 
+std::string escapeCharacters(const std::string& query)
```

Comment:
I think it's worth documenting what the escaping is being done for here (either in the function name, or as a comment), since the way escaping is performed depends on what is trying to interpret the characters / escapes. Since (I think) the intention here is to escape a regular expression pattern, you might want to call the function `escapeForRegex()` or something like that?